api/v1alpha1/apicast_types.go

```diff
@@ -227,9 +226,6 @@ type ApicastList struct {
 	Items           []Apicast `json:"items"`
 }
 
-// Ensure ApicastList implements basereconciler.ExtendedObjectList
-var _ basereconciler.ExtendedObjectList = &ApicastList{}
-
```

Comment:
I had to remove this in all API definitions as it was causing a cyclic dependency.